llmfoundry/models/mpt/modeling_mpt.py

```diff
@@ -574,14 +575,20 @@ class MPTForCausalLM(MPTPreTrainedModel):
 
     def __init__(self, config: MPTConfig):
         super().__init__(config)
-        if not config.tie_word_embeddings:
-            raise ValueError(
-                'MPTForCausalLM only supports tied word embeddings')
-
         log.info(f'Instantiating an MPTForCausalLM model from {__file__}')
 
         self.transformer: MPTModel = MPTModel(config)
 
+        self.lm_head = None
+        if config.tie_word_embeddings is False:
```

Comment:
was there a reason to do `if config.tie_word_embeddings is False:` instead of `if not config.tie_word_embeddings`:

Reply:
I do this because it seems more explicit, but 🤷‍♂️
I'll change it